specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/keys.json

```diff
@@ -1566,8 +1490,9 @@
           }
         },
         "release_policy": {
-          "$ref": "#/definitions/KeyReleasePolicy",
-          "description": "The policy rules under which the key can be exported."
+          "description": "Blob encoding the policy rules under which the key can be exported.",
```

Comment:
Given my comment above and that this same definition is repeated many times, I would actually define a data type that *should* be supported in swagger, but let the validation system check:

```json
"definitions": {
  "KeyReleasePolicy": {
    "type": "string",
    "description": "...",
    "format": "base64url"
}
```

Thing is, if you define `contentType` and/or `version`, you may want this to still be a model with those three properties and this just becomes an `object` data type.